ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,296 @@
+        borderColor = appearanceProxy.borderColor
+        disclosureIconForegroundColor = appearanceProxy.disclosureIconForegroundColor
+        iconColor = appearanceProxy.iconColor
+        leadingTextColor = appearanceProxy.labelColor
+        subtitleColor = appearanceProxy.sublabelColor
+        trailingItemForegroundColor = appearanceProxy.trailingItemForegroundColor
+
+        highlightedBackgroundColor = appearanceProxy.backgroundColor.pressed
+
+        borderSize = appearanceProxy.borderSize
+        disclosureInterspace = appearanceProxy.disclosureInterspace
+        disclosureSize = appearanceProxy.disclosureSize
+        horizontalCellPadding = appearanceProxy.horizontalCellPadding
+        iconInterspace = appearanceProxy.iconInterspace
+        iconSize = iconStyle == MSFListIconVnextStyle.large ? appearanceProxy.iconSize.large : appearanceProxy.iconSize.default
+        subtitleFont = appearanceProxy.sublabelFont
+        textFont = appearanceProxy.labelFont
+    }
+}
+
+public struct MSFListView: View {
+    @ObservedObject var state: MSFListVnextState
+    @ObservedObject var tokens: MSFListTokens
+    var cells: [MSFListVnextCellData]
+
+    public init(cells: [MSFListVnextCellData],
+                layoutType: MSFListCellVnextLayoutType,
+                iconStyle: MSFListIconVnextStyle) {
+        self.state = MSFListVnextState()
+        self.tokens = MSFListTokens(layoutType: layoutType, iconStyle: iconStyle)
+        self.cells = cells
+    }
+
+    public var body: some View {
+        List {
+            if state.sectionTitle != nil {
+                if #available(iOS 14.0, *) {
+                    Section(header: Header(title: state.sectionTitle ?? "", tokens: tokens)) {}
+                        .textCase(.none)
+                        .listRowInsets(EdgeInsets())
+                        .padding(.leading, tokens.horizontalCellPadding)
+                        .padding(.trailing, tokens.horizontalCellPadding)
+                        .background(Color.white)
+                } else {
+                    Section(header: Text(state.sectionTitle ?? "")) {}
+                }
+            }
+            ForEach(cells, id: \.self) { item in
+                MSFListCellView(cell: item, tokens: tokens)
+                    .border(state.hasBorder ? Color(tokens.borderColor) : Color.clear, width: state.hasBorder ? tokens.borderSize : 0)
+                    .fixedSize()
+            }
+        }
+        .environment(\.defaultMinListRowHeight, 0)
+        .frame(width: .infinity)
+    }
+}
+
+extension MSFListView {
+    /// View for List Cells
+    struct MSFListCellView: View {
+        var cell: MSFListVnextCellData
+        var tokens: MSFListTokens
+        @State var isTapped: Bool = false
+
+        init(cell: MSFListVnextCellData, tokens: MSFListTokens) {
+            self.cell = cell
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            HStack(spacing: 0) {
```

Comment:
I have just tried enclosing all this content in a Button and got the TableView behavior (including the background color change):

https://user-images.githubusercontent.com/68076145/103319927-6e7f2a00-49e8-11eb-9eab-d575e5e1a827.mov

...```

        var body: some View {
            Button(action: cell.onTapAction ?? {}, label: {
                HStack(spacing: 0) { ...
```